Reading_files/vascular.ipynb

```diff
@@ -1,5 +1,16 @@
 {
```

Comment:
<p>The table is nice, but this doesn't tell you much as you can't see the whole thing. Could you make a graphic instead? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Talavera-Lopez-Lab/Heart_Vascular/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>